merlin-server/constraints-dsl-compiler/src/libs/constraints-ast.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference path="./TemporalPolyfillTypes.ts"; />
```

Comment:
Do we need this hot comment in `constraints-ast.ts`?

Reply:
I guess not? I'm confused as to why it was necessary. Before this PR we would get a Not Found error if this comment was removed - it was the first thing I tried when doing this task. But if I remove it now it is fine. Now this comment isn't in constraints or scheduling.

Reply:
Actually no, this comment is required still. The tests failed, so I'm putting it back.